int8 leftMotor
int8 rightMotor